opt4j-core/src/test/java/org/opt4j/core/IndividualTest.java

```diff
@@ -8,6 +8,9 @@
 import org.opt4j.core.genotype.PermutationGenotype;
 
 public class IndividualTest {
+	
+	private boolean stateChanged = false;
```

Comment:
A proposition (not specifically targeted at this case): What do you think about making all class attributes protected instead of private. This makes writing tests a lot easier in many cases.

Reply:
SoC. Create different issue/patch.